thesheriff/domain/outlaw/outlaw_factory.py

```diff
@@ -10,6 +10,8 @@ class OutlawFactory(Outlaw):
     def create(name: str, email: str, id: Optional[int] = None) -> Outlaw:
         """Method create, produces an Outlaw instance.
 
+        :param id: Outlaw's id (optional)
```

Comment:
Could you move the id doc down so it follows the same order as in the method signature?

Reply:
ofc